Marlin/src/gcode/feature/L6470/M906.cpp

```diff
@@ -281,7 +281,7 @@ void GcodeSuite::M906() {
           break;
       #endif
 
-      #if HAS_EXTRUDERS
+      #if E_STEPPERS
```

Comment:
There should be no case where `HAS_EXTRUDERS` is not set when `E_STEPPERS > 0`. And, this code is using the E index from `get_target_extruder_from_command` which does refer (abstractly) to the 'extruder' or the 'tool index' and not to the E stepper. You can also see that `get_target_extruder_from_command` does this:
```cpp
    if (e < EXTRUDERS) return e;
```
So, if we are going to be referring here to `E_STEPPERS`, which is apparently the real case, this should not be calling `get_target_extruder_from_command` — or it should be passing a flag to limit based on `E_STEPPERS` instead of limiting based on `EXTRUDERS`.